### ISSUE TITLE
How to train a strong 1 playout network ?

### ISSUE THREAD

**Issue by lemonsqueeze** [leela-zero/leela-zero#2221]
I'm looking at #1294, it'd be great indeed to have a strong raw network.

How to train such a network ?

If i understand correctly the training pipeline ensures the network is strong at 1600 playouts, because well this is how test matches are run. Under normal circumstances if policy network is slightly off for root node it doesn't matter much, other evaluations down the tree will make up for it. At 1 playout however no such chance, hence the blunders.

Is it just a matter of running 1 playout test matches then ?
Say start from last 15x192 network, train on batch of later games, run test matches, if it doesn't beat previous network keep sampling etc.

The other question i had is how bootstrapping larger networks works:
Say when going from 20b to 40b, is the new 40b network trained from scratch on all self-play games ?
With/without test matches along the way to make sure it's improving ? Or some other magic perhaps (net2net ?)

**Comment by Ishinoshita** [comment]
If you aim at 1 playout, then using a small network might not be critical, unless you are severely constrained by memory or compute. Then, a 40b network would probably be the best choice, IMHO. So you can just use the policy of existing strong 40b.

You're talking about 'training' but I don't understand how exactly you plan to improve 1po strength over existing strong nets....

Maybe you don't need to generate new games but just to train one of the last 40b with  policy loss only, on existing recent games, with policy sharpening applied before training. However, no assurance that would yield a stronger 1po player...

**Comment by sethtroisi** [comment]
@Ishinoshita I totally agree with the policy loss only. If you were training from scratch you might want to train with value initially but then turn it off near the end of the training
